### ISSUE TITLE
About Loss

### ISSUE THREAD

**Issue by harrywang725** [xtudbxk/DSRG-tensorflow#27]
I tried to implement in pytorch , but during training , i met a problem that the BalancedSeedLoss will make softmax probability of one  in all position , and it does not work in fg area ,and the loss will keep 8~9 . Have you met this problem?
how could I solve this problem? thx.

**Comment by xtudbxk** [comment]
I didn't met the problem. And maybe you can compute the balancedseedloss for a specified image and display all the output of middle steps to see what happened.

**Comment by harrywang725** [comment]
Thx,
I used a single image to tried , I found something wrong with my  weight  initialize . When I fixed it , it's  all right .

**Comment by eyoujunjun** [comment]
I meet the same question, would you please tell me how you fix it
